spdk/jsonrpc.go

```diff
@@ -61,6 +66,7 @@ func NewClient(socketPath string) *Client {
 		transport: protocol,
 		socket:    socketPath,
 		id:        0,
+		tracer:    otel.Tracer(""),
```

Comment:
I'd probably leave a comment here why we want to use a default name from Provider here

Reply:
opened #39